plugins/pur/core/src/main/java/org/pentaho/di/repository/pur/JobDelegate.java

```diff
@@ -446,13 +422,6 @@ public DataNode elementToDataNode( final RepositoryElementInterface element ) th
 
     DataNode rootNode = new DataNode( NODE_JOB );
 
-    if ( jobMeta.getPrivateDatabases() != null ) {
-      // save all private database names http://jira.pentaho.com/browse/PPP-3413
```

Comment:
We should make sure we are not regressing on the https://hv-eng.atlassian.net/browse/PPP-3413